src/index.html

```diff
@@ -109,14 +110,18 @@ <h2>&rarr; Attend an event</h2>
       <p>The best way to get involved is to come along to one of our events.</p>
 
       <h2>&rarr; Sponsor Us</h2>
-      <p>To keep our events free, we depend on sponsors to fund food, drink and venue hire.  If you are able to help out - we'd really love to hear from you.</p>
+      <p>To keep our events free we depend on sponsors to fund food, drink and venue hire.
+        If you are able to help out we'd really love to hear from you:
+        <a href="https://twitter.com/Ox_SOH">Tweet</a> or
+        <a href="mailto:ox.summer.of.hacks@gmail.com">email</a> us.</p>
 
       <h2>&rarr; Slack</h2>
-      <p>We discuss Summer of Hacks on the <a href="slack.digitaloxford.com">digital oxford slack</a>. Come and say hi on the #soh channel.</p>
+      <p>We discuss Summer of Hacks on the <a href="slack.digitaloxford.com">Digital Oxford Slack</a>.
+        Come and say hi on the [#soh](https://digitaloxford.slack.com/messages/C17K459ND/) channel.</p>
 
       <h2>&rarr; Twitter</h2>
       <p>Some short and sweet tweets will be served from
-      <a href="https://twitter.com/Ox_SOH">our twitter account</a>.
+      <a href="https://twitter.com/Ox_SOH">our Twitter account</a>.
```

Comment:
Twitter is an app name so I've capitalised it.

> Did the same for mention of Slack a few lines up too.

Reply:
clap